docs/setup.md

```diff
@@ -345,3 +357,6 @@ gopass --yes setup --remote github.com/example/pass.git --alias example --create
 # For every other team member initialize a new store and clone the existing remote
 gopass --yes setup --remote github.com/example/pass.git --alias example --name "Jane Doe" --email "jane.doe@example.com"
 ```
+
```

Comment:
Can you remove these 3 too many lines.

Reply:
done.

Reply:
Did you push your changes? It's still the old one.

Reply:
Oops, sorry. Didn't add the changes before --amend :)